src/server/game/DataStores/DBCStores.cpp

```diff
@@ -97,7 +97,7 @@ DBCStorage <GtRegenHPPerSptEntry>         sGtRegenHPPerSptStore(GtRegenHPPerSptf
 DBCStorage <GtRegenMPPerSptEntry>         sGtRegenMPPerSptStore(GtRegenMPPerSptfmt);
 
 DBCStorage <HolidaysEntry>                sHolidaysStore(Holidaysfmt);
-
+DBCStorage <ItemEntry>                    sItemStore(Itemfmt);
```

Comment:
add a blank line above

Reply:
Gonna have to wait a week or two.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/14675/commits/8159dbc4edc680e252c4ef85847eb8bb3aefa031